### PR TITLE
WSO 410 2 step accordion brick

### PR DESCRIPTION
**Pull request by TDRoden**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php [hunk 1]
```diff
@@ -1,192 +1,199 @@
...
+              foreach ($content['field_gbl_accord_fc_content']['#items'] as $item) {
+
+                // Load the field collection item by entity id.
+                $loaded_entity = entity_load('field_collection_item', array($item['value']));
+
+                // Use array shift to get at the field collection variables without using the entity ID
+                $loaded_entity = array_shift($loaded_entity);
+
+                // If the counter is at 1, initialize variables to use for pre javascript state classes (eg. active or collapsed)
+                if ($counter == 1) {
```
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php]: Jesse would like us to use tipple equals for string typing checking.  Sometimes you might have to do === "1" if it breaks with just adding the ===. Doesn't look like you will have to here though.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php [hunk 2]
```diff
@@ -1,192 +1,199 @@
...
+                  $accordion_description = $loaded_entity->field_accordion_description[LANGUAGE_NONE][0]['value'];
+                }
+
+                // Accordion button alignment
+                if (isset($loaded_entity->field_accordion_button_alignment)) {
+                  // Define button alignment with lower case value
+                  $button_alignment = strtolower($loaded_entity->field_accordion_button_alignment[LANGUAGE_NONE][0]['value']);
+
+                  // Drop down display value is centre but css uses center
+                  if ($button_alignment == 'centre') {
```
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php]: ===

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php [hunk 3]
```diff
@@ -1,192 +1,199 @@
...
+                  }
+                }
+                else {
+                  $button_alignment = 'center';
+                }
+
+                // Classes for the checklist items depending number of items in the accordion steps field collection.
+                $fc_items_count = count($loaded_entity->field_fc_accordion_steps[LANGUAGE_NONE]);
+
+                if ($fc_items_count <= 1) {
```
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php]: === in this block

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php [hunk 4]
```diff
@@ -1,192 +1,199 @@
...
-            <?php print $main_description; ?>
-          <?php }; ?>
+          <?php
+            if ($show_heading === '0') {
+              ?>
+              <h2><?php print $main_heading; ?></h2>
+              <?php
+            }
+
+            if (isset ($main_description)) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php]: Remove extra space between isset and bracket "(isset($main_description))"

**Pull request merged**